### PR TITLE
Have a buyer be able to checkout

### PR DESCRIPTION
**Pull request by robsdagreat**
What does this PR do?

Have a buyer be able to checkout successfully placing an order

Description of Task to be completed?

After being viewing your cart and decide to checkout, a buyer is able to place an order which will be saved for next phase which is payment

How should this be manually tested?

N/A

Any background context you want to provide?

N/A

Screenshots (if appropriate)

N/A

Questions:

N/A

What are the relevant pivotal tracker stories

https://www.pivotaltracker.com/story/show/187570839

### CODE REVIEW

**Review by Ishimwe7 — APPROVED** [review]
Well done @robsdagreat .

**Review by kirengamartial — APPROVED** [review]
solve those conflicts you have otherwise you did a good job

File: src/pages/Checkout.tsx
Comment (derrick-nuby) [inline src/pages/Checkout.tsx:1]: Doesn't when a user checks out get a notification???

**Review by derrick-nuby — APPROVED** [review]
This PR is ready for merge no problem

File: src/pages/Checkout.tsx
Comment (robsdagreat) [reply src/pages/Checkout.tsx:1]: He gets an email giving the information of the order processed.

**Review by rodriguecyber — APPROVED** [review]

**Pull request merged**